tests/bluetooth/tester/overlay-le-audio.conf

```diff
@@ -15,6 +15,11 @@ CONFIG_BT_BUF_EVT_RX_COUNT=16
 CONFIG_BT_BUF_EVT_RX_SIZE=255
 CONFIG_BT_BUF_CMD_TX_SIZE=255
 
+# MICP
+CONFIG_BT_MICP_MIC_DEV=y
+CONFIG_BT_MICP_MIC_CTLR=y
+CONFIG_BT_MICP_MIC_CTLR_MAX_AICS_INST=2
```

Comment:
Should we also set `CONFIG_BT_MICP_MIC_DEV_AICS_INSTANCE_COUNT=2`? Or why do we need to support 2 AICS for the controller only?

Reply:
This is a leftover. Right now there are no tests that require more than one instance of AICS in MICP. Also added CONFIG_BT_MICP_MIC_DEV_AICS_INSTANCE_COUNT=1